Snowflake.Data.Tests/Client/AnnotatedClassForConstructorConstruction.cs

```diff
@@ -0,0 +1,22 @@
+using Snowflake.Data.Client;
+
+namespace Snowflake.Data.Tests.Client
+{
+    [SnowflakeObject(ConstructionMethod = SnowflakeObjectConstructionMethod.CONSTRUCTOR)]
```

Comment:
Its StructuredObject rather than SnowflakeObject.

Reply:
didn't we want to have all public classes to have Snowflake prefix?